monitors/SyntheticPegMonitor.js

```diff
@@ -0,0 +1,100 @@
+const { createFormatFunction, createEtherscanLinkMarkdown } = require("../common/FormattingUtils");
+
+class SyntheticPegMonitor {
+  /**
+   * @notice Constructs new Liquidator bot.
+   * @param {Object} logger Module used to send logs.
```

Comment:
why capitalize "Module"?

Reply:
Ah, I see you are capitalizing first character of first word following variable name

Reply:
can revert this if you dont like it :)